### ISSUE TITLE
steam_desktop_updater.py: error: the following arguments are required: steam_root

### ISSUE THREAD

**Issue by ghost** [gasinvein/steam-desktop-updater#4]
root@matheus-pc:/home/matheus/Downloads/steam-desktop-updater-master# python3 steam_desktop_updater.py
usage: steam_desktop_updater.py [-h] [-d DATATIR] [-c STEAM_COMMAND]
                                steam_root
steam_desktop_updater.py: error: the following arguments are required: steam_root

**Comment by gasinvein** [comment]
You're supposed to pass the path to the steam client directory, e.g. `~/.var/app/com.valvesoftware.Steam/data/Steam` for flatpak'ed Steam.

Also note that I've just updated the script to use `vdf` and `steam` modules instead of `steamfiles`, which appears to be abandoned.

**Comment by ghost** [comment]
Well, i'm having other problem:
ModuleNotFoundError: No module named 'vdf'

**Comment by gasinvein** [comment]
Yes, I wrote about this in previous message. `vdf` and `steam` python modules are required now.

**Comment by ghost** [comment]
I installed both of them, how i pass the path to the steam client directory?

**Comment by gasinvein** [comment]
As an argument for the script, e.g.
```
./steam_desktop_updater.py ~/.var/app/com.valvesoftware.Steam/data/Steam
```
